spec/nostra/account/csv_data_parser_spec.rb

```diff
@@ -0,0 +1,38 @@
+require_relative '../../../app/nostra.rb'
+
+RSpec.describe Nostra::CSVDataParser do
+  # NOTE: currently using sample from actual 1Money export
+  let(:subject) { Nostra::CSVDataParser.transactions('spec/fixture/1Money-sample.html') }
+
+  it 'returns an array of hashes' do
+    expect(subject).to be_a(Array)
+    expect(subject.count).to eql(9)
+    expect(subject.first).to be_a(Hash)
```

Comment:
I still like the practice of a single expectation per test block.  Primarily because it makes it clear which part is failing if something does fail.  Sometime there is no way to meaningfully break these things out.  Looks like this test is probably just the simple starting place, so i'm sure its not a big deal.

The other thing I'd comment on is that expectations on the class type might not be the best thing... what if you want to define a special collection/enumerable type for Transactions?  it might inherit from array, but i don't know if it would pass this test.  Same for the Hash, if you replace it with some kind of struct.

That being said, the Dry/Rom way seems to be adding more type concepts to ruby, and it might be worth finding the resilient ways of testing these sorts of "type checks"